manpages/fatlabel.8.in

```diff
@@ -23,7 +23,7 @@
 .\"
 .TH FATLABEL 8 @RELEASE_DATE@ "dosfstools @PACKAGE_VERSION@"
 .SH NAME
-fatlabel \- set or get MS-DOS filesystem label or volume ID
+fatlabel \- sets or get MS-DOS filesystem label or volume ID
```

Comment:
Looking at other man pages, for example `man ls` or `man diff`, they use first person / root form:
* `ls - list directory contents`
* `diff - compare files line by line`

so I would leave this as `set`.